### PR TITLE
docs: consortium-training terminology consistency + definition consolidation

### PR DESCRIPTION
**Pull request by ctn**
## Summary

Makes the consortium-training vocabulary consistent across docs and code, with **TAP-004** as the canonical two-phase source. Applies the "Simplicity and Consistency" principle to the open identity-level rename decision. Three commits:

## New terminology (canonical vocabulary)

These terms are now the primary vocabulary across all docs and code, defined in [TAP-004](../tech-docs/architecture/decisions/adr-004-training-loop.md) and used consistently in prose, tables, headings, diagrams, and code docstrings:

| Term | Meaning |
| :--- | :--- |
| **Consortium training** | The paradigm (umbrella). |
| **Shared-Base Loop** (Phase 1) | The collective, iterative loop that produces the Shared Base. |
| **Sovereign Build** (Phase 2) | Each member's local pipeline producing its Sovereign Model. |
| **Shared Base** | The common base model the loop produces (the "1" in N+1). No alignment. |
| **Sovereign Model** | A member's deployable, culturally-aligned model (one of the "N"). |
| **Contributed CPT** | CPT inside the loop; weights contributed back to the Shared Base. |
| **Private CPT** | CPT inside a Sovereign Build; stays local, never contributed. |
| **Stage 0/A/B/C** | Retained as the fine-grained breakdown *inside* the Sovereign Build (anchored in TAP-005), with Stage A explicitly mapped to Contributed/Private CPT. |

**Renames applied:** "Sovereign Model Pipeline" → "The Sovereign Build" (TAP-005 title + link labels + index); "global base" / "global weights" → "Shared Base"; "federated training" → "consortium training" (including ~20 PRD references).

**Kept as-is (identity-level names, not loose prose):** TAP-004 ADR title ("The Consortium Training Loop"), SVG `<title>` elements, and paradigm-level "consortium training loop" references in governance/VISION docs that reference the paradigm rather than describing the mechanism.

### 1. Terminology + definition consolidation + Mermaid (`707ef8a`)
- **Terminology (Part A):** Retitle TAP-005 "Sovereign Model Pipeline" → "The Sovereign Build" (ADR title, `decisions/README` index row, `training-approaches` link label, adr-005 body prose). TAP-004 title and SVG internal titles kept as-is.
- **Definition consolidation (Part B):** TAP-002 is now the single canonical definition; verbatim re-statements removed from `ARCHITECTURE.md`, `training-approaches.md`, and `5-architectural-options.md` (replaced with glosses + links to TAP-002).
- **Mermaid diagrams (Part C):** Progressive-disclosure diagrams with a shared palette consistent with TAP-004, added to adr-002, adr-005, `ARCHITECTURE.md` (replacing the ASCII-art loop), and `training-approaches.md`.

### 2. SVG diagram alignment (`f56a33a`)
Bring the six canonical architecture SVGs into line with the two-phase model and the shared Mermaid palette.
- **`consortium-n-plus-1.svg`** redrawn — the old version conflated Phase 1 and Phase 2 (nodes ran the full Stages 0–C pipeline *and* contributed Stage A weights back). New version shows the two phases as distinct, hinged on the Shared Base.
- **`consortium-training-loop.svg`** relabeled to canonical terms (Contributed CPT, Shared Base, weight vectors, coordinator merge); step 1 marked as one-time seed.
- **`core-plus-sovereign-stack.svg`** — Stage B/C labels were reversed; now B = SFT, C = alignment (matches TAP-005).
- All six SVGs recolored to the Mermaid palette so the same concept reads in the same color across SVG and Mermaid (sovereign green→purple, coordinator blue→teal, base blue standardized).
- `diagrams/README.md` index: `consortium-training-loop.svg` is no longer embedded by TAP-004 (it uses Mermaid); row now lists `training-approaches.md` only.

### 3. PRD reconciliation (`d7f9d0f`)
The PRD predated the two-phase rewrite and described the training mechanism as "federated gradient sharing" at "1,000-node" edge scale with a 1/3-Byzantine bound — all of which the ratified ADRs explicitly reject. Reconciled to the ADRs (per the project rule: ADRs win).
- Terminology sweep: all ~20 "federated" refs → "consortium training."
- Substantive mechanism reconciliations (Must requirements): **SR-12** (gradient sharing → weight-vector contribution + secure aggregation + optional DP), **SR-13** (blanket Byzantine 1/3 → robustness to the failure modes the collaborative trust model admits), **AR-1** (gradient-sync protocol → Shared-Base Loop protocol with pluggable aggregation), **AR-6** (1,000-node scale → target-membership scale; Sovereign Builds unbounded), **G5** (1,000+ nodes / 5-min gradient sync → 7 G7 members scaling to a global consortium).
- Header bumped to v1.1 (2026-06-18) with a reconciliation note citing TAP-002/TAP-004.

## Test plan

- [x] Canonical `rg` terminology verification command passes (every remaining match is an ADR title/link label, an intentional old→new bridge mapping, or an explicitly-kept residual).
- [x] Definition fingerprint grep confirms one canonical definition (adr-002) plus one sanctioned gloss (`5-architectural-options.md` → TAP-002).
- [x] All six touched SVGs validate as well-formed XML.
- [ ] Mermaid diagrams render correctly on the docs site / GitHub.

## Out of scope

- TAP-004 ADR title, SVG `<title>` elements, and paradigm-level "consortium training loop" references in governance/VISION docs — kept as-is (identity-level names, not loose prose).
- `0-tva-methodology.md:17` uses different wording for distinct methodology-framing work — not a duplicate, left alone.

Made with [Cursor](https://cursor.com)

### CODE REVIEW

File: src/tapestry/training/consortium/__init__.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 """Consortium-training proof of concept for Project Tapestry.
 
-This package models the Tapestry-specific training loop described in the ADRs:
+This package models the Tapestry-specific Shared-Base Loop described in the ADRs:
```
Comment (deanwampler) [inline src/tapestry/training/consortium/__init__.py:3]: I know it makes it more wordy, but how about "shared-base training loop", since training loop is probably more recognizable to people, while shared-based emphasizes the context?

Comment (deanwampler) [reply src/tapestry/training/consortium/__init__.py:3]: I suggest as a follow up PR, that we add a glossary in `tech-docs` that defines the terms in one place, e.g., "Shared Base Loop", "Contributed CPT", etc.

File: tech-docs/architecture/decisions/adr-002-consortium-training.md [hunk 2]
```diff
@@ -28,11 +42,11 @@ This is distinct from federated learning (designed for millions of small edge cl
...
-| **What crosses the network** | N/A — internal interconnect | FedSGD: per-step gradients; FedAvg: local model weight vectors after local training | Local model weight vectors after Stage A CPT |
+| **What crosses the network** | N/A — internal interconnect | FedSGD: per-step gradients; FedAvg: local model weight vectors after local training | Local model weight vectors after Contributed CPT |
 | **Communication cadence** | Every step; fast interconnect | Varies by method and deployment | Operational choice — frequent (cluster-like) or infrequent (geo-distributed) |
 | **Model scale** | Frontier | Typically small to medium | Frontier |
 | **Governance** | Single owner decides all | Aggregator-driven; clients have no architectural voice | Consortium with shared ownership and governance rights |
-| **Each node's outcome** | N/A — one model | Same global model (or personalized variant in PFL) | Sovereign model: shared base + community-specific alignment |
+| **Each node's outcome** | N/A — one model | Same global model (or personalized variant in PFL) | Sovereign Model: Shared Base + community-specific Sovereign Build |
```
Comment (deanwampler) [inline tech-docs/architecture/decisions/adr-002-consortium-training.md:50]: I think we definitely need to be clear, perhaps at least in my glossary suggestion above, that a "sovereign build" is "post-trained" from the "shared base".

File: tech-docs/architecture/decisions/adr-004-training-loop.md [hunk 3]
```diff
@@ -5,71 +5,141 @@
...
-- Data never leaves the node — only model weight vectors after local CPT cross the wire (DG2).
-- Each cycle improves the global model with sovereign knowledge from all nodes, creating a virtuous cycle: the next round of continued pretraining starts from a better base.
-- The iterative nature means the 80/20 ratio (base vs. sovereign) shifts naturally over time as the global model absorbs more sovereign knowledge.
+Given the core-plus-sovereign architecture ([TAP-001](adr-001-core-plus-sovereign.md)) and the consortium training model ([TAP-002](adr-002-consortium-training.md)), this ADR defines how training actually flows — and separates the **two distinct phases** that earlier drafts blurred together.
+
+## The two phases
+
+Consortium training has two phases, run in order:
+
+1. **Shared-Base Loop** — collective and iterative. Members improve one shared base model together. Its output is a *base*, not a deployable product, and it carries **no cultural alignment**.
```
Comment (deanwampler) [inline tech-docs/architecture/decisions/adr-004-training-loop.md:19]: Yes, this is the clarification I was looking for above. Rather than a glossary, an alternative is to link to this section of this doc.

File: tech-docs/architecture/decisions/adr-004-training-loop.md [hunk 4]
```diff
@@ -5,71 +5,141 @@
...
+2. **Sovereign Build** — local and per-member. Each member takes the shared base and builds its own deployable model, including alignment. Nothing here is shared back.
```
Comment (deanwampler) [inline tech-docs/architecture/decisions/adr-004-training-loop.md:20]: Should we use the term "node", since we have adopted that elsewhere. Personally, I don't like the term (years of exposure makes me think of compute _node_ in a cluster...), but as long as we are consistent. Or do you mean "member" as in the org and I should think of "node" as the compute environment the member provides?

File: tech-docs/architecture/decisions/adr-002-consortium-training.md [hunk 5]
```diff
@@ -17,6 +17,20 @@ See [`training-approaches.md`](../../reference/training-approaches.md) for a ful
 
 Tapestry uses **consortium training**: a small number of large, trusted, heterogeneous nodes collaboratively training a shared model, where data sovereignty is a first-order architectural constraint and cultural alignment is the goal.
 
+```mermaid
+flowchart LR
+  LOOP["Shared-Base Loop<br/>(N members · Contributed CPT)"]:::collective
+  SB[("Shared Base<br/>the 1")]:::base
```
Comment (deanwampler) [inline tech-docs/architecture/decisions/adr-002-consortium-training.md]: How about putting quotes around `1` here, as in `the '1'`? I suggest this because when I looked at the diagram, I saw `the 1` and thought at first it was a typo, until I remembered the `N + 1` terminology.

File: tech-docs/architecture/decisions/adr-004-training-loop.md (hunk 4 above)
Comment (deanwampler) [reply tech-docs/architecture/decisions/adr-004-training-loop.md:20]: Side note: I think we would probably explore applying "some" cultural alignment in the base stage, to see how well it works and how far you can go, but it might not work well at all, especially across cultures. Either way, I think it's fine to state here that none will be done to keep the text clearer.

File: tech-docs/architecture/decisions/adr-002-consortium-training.md [hunk 6]
```diff
@@ -17,6 +17,20 @@ See [`training-approaches.md`](../../reference/training-approaches.md) for a ful
```
Comment (deanwampler) [inline tech-docs/architecture/decisions/adr-002-consortium-training.md:18]: Add a `revised` field in the table at the top, like you did for the 004 ADR.

File: tech-docs/architecture/decisions/adr-005-sovereign-pipeline.md [hunk 7]
```diff
@@ -1,4 +1,4 @@
-# TAP-005: Sovereign Model Pipeline
+# TAP-005: The Sovereign Build
 
 | Field | Value |
 | :---- | :---- |
```
Comment (deanwampler) [inline tech-docs/architecture/decisions/adr-005-sovereign-pipeline.md:4]: Add a `revised` field in the table at the top, like you did for the 004 ADR.

File: tech-docs/architecture/decisions/adr-006-phased-base-model.md [hunk 8]
```diff
@@ -33,7 +33,7 @@ Sovereign contributions (continued pretraining outputs, alignment layers, domain
 ## Rationale
```
Comment (deanwampler) [inline tech-docs/architecture/decisions/adr-006-phased-base-model.md:34]: Add a `revised` field in the table at the top, like you did for the 004 ADR. (This is not a major change, though.)

**Review by deanwampler — APPROVED** [review]
Looks good. I made some comments, most of which are just to think about. The only change I would seriously consider making is to add `revised` rows to every ADR that is changed, like you did for `004`. I'll approve the PR and let you decide if and when you want to do that before merging.

File: src/tapestry/training/consortium/__init__.py (hunk 1 above)
Comment (ctn) [reply src/tapestry/training/consortium/__init__.py:3]: Declining the "shared-base training loop" rename, intentionally — it would reintroduce a synonym for the canonical term (Shared-Base Loop) this PR just standardized everywhere; the whole point of the pass is to collapse synonyms, not add one. Happy to revisit in a separate decision if you want to argue for it as the *primary* name.

On the glossary: agreed, deferring to a follow-up PR (your own suggestion). The per-ADR definitions plus the terminology table in this PR's description are the interim single-source-of-truth.

File: tech-docs/architecture/decisions/adr-002-consortium-training.md (hunk 2 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-002-consortium-training.md:50]: Agreed — "a Sovereign Build is post-trained from the Shared Base" is exactly the relationship TAP-004 establishes (Phase 2 takes the Phase-1 output and post-trains it). I'll make sure the glossary entry states that explicitly when I do the follow-up glossary PR.

File: tech-docs/architecture/decisions/adr-004-training-loop.md (hunk 3 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-004-training-loop.md:19]: Agreed — a short cross-link is lighter-weight than a glossary and gets the reader to the definition in one hop. I'll fold cross-links in as part of the follow-up glossary PR so they're done once, consistently, rather than piecemeal here.

File: tech-docs/architecture/decisions/adr-004-training-loop.md (hunk 4 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-004-training-loop.md:20]: Flagging this one back to you rather than acting silently — it's a real conceptual choice, not a think-about. TAP-002's definition says "nodes"; TAP-004 deliberately says "members." Your read (member = the org; node = the compute environment the member provides) is how I'd reconcile it, but picking one as primary and sweeping the docs is a scope expansion I don't want to do under a terminology-consistency pass without a call. Want me to open a follow-up issue?

On the base-stage alignment side note: leaving the text as-is per your read. "No alignment in the base" is the clear baseline; exploring partial base-stage alignment can be its own ADR if it proves worth pursuing.

File: tech-docs/architecture/decisions/adr-002-consortium-training.md (hunk 5 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-002-consortium-training.md]: Done in `001d5c8` — quoted as `the '1'` in both the adr-002 Mermaid and the `consortium-n-plus-1.svg` (was curly `the “1”` in the SVG) so it reads as the N+1 terminology, not a typo, consistently across both.

File: tech-docs/architecture/decisions/adr-002-consortium-training.md (hunk 6 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-002-consortium-training.md:18]: Done in `001d5c8` — added a `Revised` field-row (Jun 19, 2026) matching TAP-004's format/placement.

File: tech-docs/architecture/decisions/adr-005-sovereign-pipeline.md (hunk 7 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-005-sovereign-pipeline.md:4]: Done in `001d5c8` — added a `Revised` field-row matching TAP-004's format. Also added one to TAP-006 and TAP-007 (the other changed ADRs that lacked it).

File: tech-docs/architecture/decisions/adr-006-phased-base-model.md (hunk 8 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-006-phased-base-model.md:34]: Done in `001d5c8` — added a `Revised` field-row (Jun 19, 2026 — terminology aligned to the two-phase model). Agree it's not major; kept the note short.

**Comment by ctn** [comment]
Follow-up: the in-repo glossary I mentioned here is now PR #80 — https://github.com/The-AI-Alliance/tapestry/pull/80. It defines the consortium-training terms (plus Member, Coordinator, N+1, Cultural Alignment) in `tech-docs/reference/glossary.md`, with cross-links back from adr-002/004/005, ARCHITECTURE.md, training-approaches.md, and the PRD. @deanwampler

File: tech-docs/architecture/decisions/adr-004-training-loop.md (hunk 4 above)
Comment (ctn) [reply tech-docs/architecture/decisions/adr-004-training-loop.md:20]: Resolved in PR #80 (`ab75909`), adopting the role distinction you proposed:

- **Member** = the participating organization (governance voice, sovereignty rights, exit rights).
- **Node** = the compute environment a member operates (where data resides, where training runs, what disconnects in island mode).

This is additive rather than a sweep: most "node" usages in the docs already mean the compute environment (`data stays on the node`, `node operates under national law`, `sovereign node may disconnect`, `each node receives the checkpoint`) and stay correct. Only the handful where "node" was misused to mean the organization were fixed — adr-002 (3 spots), ARCHITECTURE.md (3 spots), training-approaches.md, 5-architectural-options.md, 3-value-propositions.md.

Compound usages (`member node`, `sovereign node`, `participating nodes`) are retained — they now read correctly under the distinction.

The glossary has firm Member and Node entries with cross-refs. Happy to adjust if the split doesn't match your intent.

**Comment by deanwampler** [comment]
All good. Thanks!
